docs/index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   external-scaler-azure-cosmos-db:
```

Comment:
Can you revert changes in this file and remove `keda-2.6.3.tgz` please?

Reply:
reverted